lib/src/console/command/set_command.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:cider/src/console/command/application_command.dart';
+import 'package:cider/src/console/console.dart';
+import 'package:cider/src/console/exit_code.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+class SetCommand extends ApplicationCommand {
+  SetCommand(this._console);
+
+  final Console _console;
+
+  @override
+  final description = 'Setting given version explicitly';
+
+  @override
+  final name = 'set';
+
+  @override
+  final aliases = ['switch', 'change'];
+
+  @override
+  int run() {
+    try {
+      if (argResults.rest.isEmpty) {
+        throw UnsupportedError('Version must be specified');
+      } else if (argResults.rest.length > 1) {
+        throw UnsupportedError('Too many arguments passed');
+      } else {
+        Version version;
+        try {
+          version = Version.parse(argResults.rest.single);
+        } on dynamic catch (_) {
```

Comment:
I think `Version.parse` throws `FormatException` in this case. What's the reason for using `dynamic`?

Reply:
Good point, I didn't know about the type of error, so I did dynamic.